BIOS_Automation/BIOS_parameters_management/Process_BIOS_parameters_fixing/Tasks/Task_Create_managed_entity.php

```diff
@@ -13,8 +13,15 @@
 	$device_external_reference = $context['device_external_reference'] = hash('crc32', md5($context['server_vendor'].$context['server_ip_address'].$context['server_mac_address']));
 	
 	//model and manufacture IDs should be changed properlly
-	$manufacture_id = $context['manufacture_id'] = '191200';
-	$model_id = $context['model_id'] = '191200';
+    if ($context['mgmt_interface'] == 'REDFISH') {
```

Comment:
Since we have only single vendor (REDFISH) we do not needed from my perspective. Anyway, it is demo, not production ready code. It is easier to keep it inside code.